horace_core/sqw/@SQWDnDBase/IX_dataset_2d.m

```diff
@@ -1,59 +1,21 @@
 function wout = IX_dataset_2d (w)
-% Convert 2D sqw object into IX_dataset_2d
+% Convert 2D sqw object(s) into IX_dataset_2d(s)
```

Comment:
and dnd's?

Reply:
dnd overloads the function and does the job and sqw function delegates job to dnd